lib/internal/readline/utils.js

```diff
@@ -17,6 +17,7 @@ const kEscapeCodeTimeout = 500;
 
 const kKeypressDecoder = Symbol('keypress-decoder');
 const kEscapeDecoder = Symbol('escape-decoder');
+const kMinSurrogateCodePoint = 0x10000; // 2 ** 16
```

Comment:
Hm … without looking at the definition, I’d have guessed from the name that this would be `0xD800`, i.e. the minimum UTF-16 surrogate code unit; how about `kUTF16SurrogateThreshold`? (Still not great but it’s hard to come up with something good)